File: VB/WpfApplication1/WizardCustomization/CustomSelectColumnsPage.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Data.XtraReports.ServiceModel
Imports DevExpress.Data.XtraReports.Wizard
Imports DevExpress.Data.XtraReports.Wizard.Presenters
Imports DevExpress.Data.XtraReports.Wizard.Views
Imports System
' ...

Namespace WpfApplication1.WizardCustomization
	Friend Class CustomSelectColumnsPage
		Inherits DevExpress.Data.XtraReports.Wizard.Presenters.SelectColumnsPage(Of ReportModel)
		Public Sub New(ByVal view As ISelectColumnsPageView, ByVal client As IReportWizardServiceClient, ByVal cache As IColumnInfoCache)
			MyBase.New(view, client, cache)
		End Sub

		Public Overrides Function GetNextPageType() As Type
			Return GetType(ChooseReportLayoutPage(Of ReportModel))
		End Function
	End Class
End Namespace

File: CS/WpfApplication1/WizardCustomization/CustomSelectColumnsPage.cs
using DevExpress.Data.XtraReports.ServiceModel;
using DevExpress.Data.XtraReports.Wizard;
using DevExpress.Data.XtraReports.Wizard.Presenters;
using DevExpress.Data.XtraReports.Wizard.Views;
using System;
// ...

namespace WpfApplication1.WizardCustomization {
    class CustomSelectColumnsPage : DevExpress.Data.XtraReports.Wizard.Presenters.SelectColumnsPage<ReportModel> {
        public CustomSelectColumnsPage(
            ISelectColumnsPageView view, 
            IReportWizardServiceClient client, 
            IColumnInfoCache cache)
            : base(view, client, cache) { }

        public override Type GetNextPageType() {
            return typeof(ChooseReportLayoutPage<ReportModel>);
        }
    }
}
